static_src/stores/base_store.js

```diff
@@ -9,6 +9,10 @@ import Immutable from 'immutable';
 import AppDispatcher from '../dispatcher.js';
 import LoadingStatus from '../util/loading_status.js';
 
+// TODO clean up the listeners and reduce the MAX_LISTENERS to 15
+const MAX_LISTENERS = 20;
+const MAX_LISTENERS_THRESHOLD = 10;
```

Comment:
Yea that's the pattern I've been going for recently, there are some old small components that don't follow it.